### PR TITLE
CLOUDP-93391: Retrieve details for one unaccepted invitation to the specified Atlas project.

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-93391

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli iam project invitation get 60c884d12dbba167ddbe650c
ID                         USERNAME                      CREATED AT             EXPIRES AT
60c884d12dbba167ddbe650c   ahmed.hamed-aly@mongodb.com   2021-06-15T10:45:37Z   2021-07-15T10:45:37Z
```
```
➜  ~/workspace/mongoCLI git:(CLOUDP-93391) ✗ ./bin/mongocli iam project invitation get 60c884d12dbba167ddbe650c -o json
{
  "id": "60c884d12dbba167ddbe650c",
  "groupId": "5e4e593f70dfbf1010295836",
  "groupName": "CLI ANDREA",
  "createdAt": "2021-06-15T10:45:37Z",
  "expiresAt": "2021-07-15T10:45:37Z",
  "inviterUsername": "andrea.angiolillo@mongodb.com",
  "username": "ahmed.hamed-aly@mongodb.com",
  "roles": [
    "GROUP_OWNER"
  ]
}
```

### CODE REVIEW

File: internal/cli/iam/projects/invitations/list_test.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func TestList_Run(t *testing.T) {
 
 	mockStore.
 		EXPECT().
-		ProjectInvitations(listOpts.ConfigOrgID(), listOpts.newInvitationOptions()).
+		ProjectInvitations(listOpts.ConfigProjectID(), listOpts.newInvitationOptions()).
```
Comment (gssbzn) [inline internal/cli/iam/projects/invitations/list_test.go:40]: 👀  can you please start adding e2e testing, I think all of org invites is testable now, and listing should testable as well, even if it's to return an empty list

Somebody added this file and I think we have forgotten about it https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md

File: internal/cli/iam/projects/invitations/describe.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+// mongocli iam project(s) invitations describe|get <ID> [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
+	opts := new(DescribeOpts)
+	opts.Template = describeTemplate
+	cmd := &cobra.Command{
+		Use:     "describe <ID>",
+		Aliases: []string{"get"},
+		Args:    require.ExactArgs(1),
+		Short:   "Retrieve details for one pending invitation to the specified Atlas project.",
```
Comment (gssbzn) [inline internal/cli/iam/projects/invitations/describe.go:61]: let's not mention atlas

File: internal/cli/iam/projects/invitations/list_test.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/iam/projects/invitations/list_test.go:40]: I have created this task for the e2e tests https://jira.mongodb.org/browse/CLOUDP-93856. I was planning to add them once I have added also the project commands.

Comment (gssbzn) [reply internal/cli/iam/projects/invitations/list_test.go:40]: we should not wait long between adding commands and adding e2e tests, I sometimes forget to check and let them slide as it's hard to test create without a delete in place

**Review by gssbzn — APPROVED** [review]
LGTM but something I had forgotten and I'm documenting now is that since we are about to start autogenerating docs we'll start asking for review from the docs team on changes that have user facing copy

**Pull request merged**